src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -563,6 +571,13 @@ def pollSubscriptions(self):
 
 
                 # Assemble a dict of all the info
+                inputDataset = None
+                locations = None
+                dbsurl = wmTask.inputDatasetDBSURL()
```

Comment:
This is where your problem comes in. wmTask in the unit test has that weird string for DBS which is invalid. I guess the changes here are related to the changes for Condor? 

I'll keep looking, but I'm trying to understand why you added this code in the first place.

I suspect just changing the string will make everything go smoothly.